docs/content/how-to/kubevirt/configuring-network.md

```diff
@@ -58,3 +58,72 @@ hcp create cluster kubevirt \
 --additional-network name:my-namespace/network1 \
 ```
 
+## Avoiding Network CIDR Collisions between Hosting and Guest Clusters.
+
+When creating a guest cluster, it's crucial to ensure that the hosting and guest clusters do not share the same subnet (cidr) for the cluster network.
+This prevents DNS-related conflicts and avoids issues during the deployment of guest clusters like below.
+
+```
+#oc get hostedclusters.hypershift.openshift.io -n clusters
+NAME      VERSION   KUBECONFIG                 PROGRESS   AVAILABLE   PROGRESSING   MESSAGE
+testing             testing-admin-kubeconfig   Partial    True        False         The hosted control plane is available
+
+#oc get co
+NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
+console                                    4.15.8    False       True          True       29m     DeploymentAvailable: 0 replicas available for console deployment...
+.
+.
+ingress                                    4.15.8    True        False         True       28m     The "default" ingress controller reports Degraded=True: DegradedConditions: \
+ One or more other status conditions indicate a degraded state: CanaryChecksSucceeding=False (CanaryChecksRepetitiveFailures: Canary route checks for the default ingress controller are failing)
+
+
+#oc logs console-686c5754f8-2x55h
+
+W0410 08:54:43.523348       1 authoptions.go:99] Flag inactivity-timeout is set to less then 300 seconds and will be ignored!
+F0410 08:54:47.551180       1 authoptions.go:286] Error initializing authenticator: failed to construct OAuth endpoint cache: failed to setup an async cache - caching func returned \
+error: request to OAuth issuer endpoint https://oauth-clusters-testing.apps.example.com:443/oauth/token failed: Head "https://oauth-clusters-testing.apps.example.com:443": dial tcp: \
+lookup oauth-clusters-testing.apps.example.com on <MASKED IP Address>:53: server misbehaving
+```
```

Comment:
For now, I think i'd recommend avoiding documentation around how to detect that the issue is occurring. These log messages can occur due to other reasons besides cidr range conflicts.

i'm working on a condition we can use to help people understand more accurately when a collision has occurred.

For now, maybe we can document how to avoid the collision.  I'd focus on two aspects.
1. How to detect the existing cluster network cidr range from the underlying infrastructure cluster.
2. How to tell if the HCP default cidr range conflicts with the infra cidr.
3. How to use the `hcp` cli to set a custom cluster cidr range for the guest cluster in the event that a collision does occur.

It's worth pointing out that the default cidr range used by HCP is meant to be offset from the default standalone OCP cidr range, so under normal operation, this shouldn't be an issue for people.

Reply:
@davidvossel Updated, Please take a look at the latest version.